R/bds.R

```diff
@@ -59,3 +59,14 @@ bds <- function(security, field, options=NULL,
     bds_Impl(con, security, field, options, overrides, verbose, identity)
 }
 
+
+bds_debug <- function(security, field, options=NULL,
```

Comment:
As above, I suggest to remove.  Keep the branch if you will or a copy but we do not this merged.